crates/typst-library/src/text/lang.rs

```diff
@@ -14,7 +14,7 @@ macro_rules! translation {
     };
 }
 
-const TRANSLATIONS: [(&str, &str); 38] = [
+const TRANSLATIONS: [(&str, &str); 39] = [
```

Comment:
```suggestion
const TRANSLATIONS: &[(&str, &str)] = [
```

If the MSRV allows it, this could be done to prevent having to change this number every time a new translation is added.

Reply:
sorry, may I ask what should I do next? this is my first time submitting translation. I'm assuming I just need to click the commit suggestion button, but I don't know what the Minimum Supported Rust Version is about and what should I do about it.

Reply:
You can commit the suggestion, and if GitHub tells you the tests now fails, just revert it back to what it was by making a new commit.

But this is not necessary, just a small improvement that can be done later, so don't worry if you are scared to break something, there is no express need to do that.

Reply:
okay, I'll try commit the suggestion.